src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java

```diff
@@ -69,6 +82,111 @@ public List<CaseEvent> handledEvents() {
         return EVENTS;
     }
 
+    // This is currently a mid event but once pre states are defined it should be moved to an about to start event.
+    // Once it has been moved to an about to start event the following file will need to be updated:
+    // FlowStateAllowedEventService.java.
+    // This way pressing previous on the ccd page won't end up calling this method again and thus
+    // repopulating the fields if they have been changed.
+    // There is no reason to add conditionals to avoid this here since having it as an about to start event will mean
+    // it is only ever called once.
+    // Then any changes to fields in ccd will persist in ccd regardless of backwards or forwards page navigation.
+    private CallbackResponse prePopulateDisposalHearingPage(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        CaseData.CaseDataBuilder updatedData = caseData.toBuilder();
+
+        DisposalHearingJudgesRecital tempDisposalHearingJudgesRecital = DisposalHearingJudgesRecital.builder()
+            .input("Upon considering the claim Form and Particulars of Claim/statements of case"
+                       + " [and the directions questionnaires] \n\nIT IS ORDERED that:-")
+            .build();
+
+        updatedData.disposalHearingJudgesRecital(tempDisposalHearingJudgesRecital).build();
+
+        DisposalHearingDisclosureOfDocuments tempDisposalHearingDisclosureOfDocuments =
+            DisposalHearingDisclosureOfDocuments.builder()
```

Comment:
the formatting here is due to checkstyle having a 120 character limit on its lines, other formatting in the file will also be due to the same reason.